cogs/games.py

```diff
@@ -1,12 +1,98 @@
 import discord
 from discord.ext import commands
+import tictactoe
+import random
+import asyncio
+import datetime
 import asyncpg
 
+
+ongoing_ttt = False
```

Comment:
Going through the code, I think the `ongoing_ttt` variable will not be unique to each server (if it gets set to True because of a game in one server, it'll reflect for _all_ servers) which would mean that there'd be one game only at a time through all servers
Needs to be tested but I think that's how it works.

Reply:
Alright I'll test that today and then think about it.

Reply:
alright so it works like you said it would. Need to come up with a solution now